### PR TITLE
task/switch-1683 update contract deployment script

### PR DESCRIPTION
**Pull request by nichonien**
### Summary

|             |   |
|-------------|---|
| Description | Update truffle config and contract migration script for deployment  |
| Jira issue  |  https://energyweb.atlassian.net/browse/SWTCH-1683  |

### CODE REVIEW

**Review by JGiter — COMMENTED** [review]
Since we are going to deploy with `truffle migrate` in proxyidentity can we remove https://github.com/energywebfoundation/ew-did-registry/blob/development/scripts/deployIdentityManager.js?

**Comment by nichonien** [comment]
> Since we are going to deploy with `truffle migrate` in proxyidentity can we remove https://github.com/energywebfoundation/ew-did-registry/blob/development/scripts/deployIdentityManager.js?

Hmm, nice point. I will remove it.

**Review by JGiter — APPROVED** [review]

**Comment by nichonien** [comment]
@jrhender @whitneypurdum I see the `snyk` test failing but I cannot know why, it says I don't have access though I was able to see the results in my previous PR.

File: packages/proxyIdentity/migrations/2_identityManager_migrations.js [hunk 1]
```diff
@@ -2,10 +2,13 @@
 const { deployProxy } = require('@openzeppelin/truffle-upgrades');
 
 // needs to be updated with actual address
-const libraryAddress = '0x4B0897b0513fdC7C541B6d9D7E929C4e5364D2dB';
+const libraryAddress = '0x4c392f00a17B1b77aB059E6dc9ccE91F74113c66';
```
Comment (jrhender) [inline packages/proxyIdentity/migrations/2_identityManager_migrations.js:5]: @nichonien is this comment of `// needs to be updated with actual address` still accurate? What contract is `'0x4c392f00a17B1b77aB059E6dc9ccE91F74113c66'`?

Comment (nichonien) [reply packages/proxyIdentity/migrations/2_identityManager_migrations.js:5]: That is correct address for the latest deployment. This comment is from the standpoint when someone else uses this script, this address would be irrelevant at that point of time for a fresh deployment. Maybe I should rephrase this comment?

Comment (nichonien) [reply packages/proxyIdentity/migrations/2_identityManager_migrations.js]: Updated the comment.

**Review by jrhender — APPROVED** [review]
The failing Snyk check for licenses, which I think we still need to adjust. So this is good from my perspective.

**Pull request merged**